packages/pebble-web/src/components/Select.tsx

```diff
@@ -156,7 +156,7 @@ function Select<OptionType>(props: SelectProps<OptionType>) {
               <OptionGroupRadio
                 selected={props.selected}
                 onChange={(_value, extras) => {
-                  if (_value) props.onChange(_value, extras);
+                  if (_value !== undefined) props.onChange(_value, extras);
```

Comment:
I think this was on purpose for "de-selecting". Not sure, though.

Reply:
currently no logic to deselect is present, just want other nullish values to pass